addons/web/static/src/views/view_dialogs/export_data_dialog.xml

```diff
@@ -31,90 +31,91 @@
+                                        exportList="state.exportList"
+                                        isFieldExpandable="isFieldExpandable"
+                                        isDebug="isDebug"
+                                        onClick.bind="onToggleExpandField"
+                                        onAdd.bind="onAddItemExportList"
+                                    />
                                 </t>
-                                <option class="font-italic" value="new_template">New template </option>
-                            </select>
-                        </t>
-                        <t t-if="state.isEditingTemplate">
-                            <button t-if="state.templateId === 'new_template'" type="button" class="btn btn-secondary ml-1 o_save_list_btn" t-on-click.stop="onSaveExportTemplate">
-                                <i class="fa fa-floppy-o" />
-                            </button>
-                            <button type="button" class="btn btn-secondary ml-1 o_cancel_list_btn" t-on-click.stop="onCancelExportTemplate">
-                                <i t-attf-class="fa {{ state.templateId === 'new_template' ? 'fa-times' : 'fa-undo' }}" />
-                            </button>
-                        </t>
-                        <t t-else="">
-                            <button t-if="state.templateId" type="button" class="btn btn-secondary ml-1 o_delete_exported_list" t-on-click.stop="onDeleteExportTemplate">
-                                <i class="fa fa-trash" />
-                            </button>
-                        </t>
+                            </t>
+                            <h3 t-else="" class="text-center text-muted mt-5 o_no_match">No match found.</h3>
+                        </div>
                     </div>
                 </div>
-                <div class="o_right_field_panel h-100 px-2 overflow-auto border">
-                    <ul class="o_fields_list list-unstyled" t-ref="draggable">
-                        <t t-foreach="state.exportList" t-as="field" t-key="field.name">
-                            <li class="o_export_field" t-att-data-field_id="field.name">
-                                <span class="fa fa-sort o_sort_field mx-1" t-attf-style="opacity:{{ state.exportList.length === 1 ? 0 : 1 }}" />
-                                <span t-if="isDebug and field.id" t-esc="`${field.string or field.label} (${field.id})`" />
-                                <span t-else="" t-esc="field.string or field.label" />
-                                <span class="fa fa-trash m-1 float-end o_remove_field o_cursor_pointer" t-att-title="removeFieldText" t-on-click.stop="onRemoveItemExportList" />
-                            </li>
+                <div t-attf-class="o_right_panel col-12 col-md-6 {{ state.isDraggable ? 'h-50' : 'h-100' }} d-flex flex-column flex-nowrap mt-3 mt-md-0">
```

Comment:
env.isSmall + useState(env) ?

Reply:
I tried this way but couldn't make it work. It was not reactive and didn't rerendered the dialog to adapt the UI